mec/cmd/mec/main.go

```diff
@@ -95,7 +94,10 @@ func createCommand(args []string) *cobra.Command {
 				log.Errorf("Failed to create OSSMPullStrategy: %v", err)
 			}
 			w := server.NewWorker(config, p, baseURL, serveDirectory)
-
+			if w == nil {
```

Comment:
probably not on the scope of this pr, but we could switch to `RunE` and return proper errors.

Reply:
I made it return an error for now